src/data/write_init_files.py

```diff
@@ -47,7 +47,8 @@
 #Main function
 ##############
 
-def write_init_files(cap_database, ic_names, registry_constituents, outdir,
+def write_init_files(cap_database, ic_names, registry_constituents, vars_init_value,
+                     outdir,
                      file_find_func, source_paths, indent, logger,
                      phys_check_filename=None, phys_input_filename=None):
```

Comment:
The spacing looks a little strang here, maybe do something like this instead (?):

```suggestion
def write_init_files(cap_database, ic_names, registry_constituents, vars_init_value,
                     outdir, file_find_func, source_paths, indent, logger,
                     phys_check_filename=None, phys_input_filename=None):
```